### ISSUE TITLE
Upgrade help-me to v5.0.0 to resolve Missing Release of Resource after Effective Lifetime

### ISSUE THREAD

**Issue by jackbridger** [mqttjs/MQTT.js#1770]
- The dependency help-me recently released [5.0.0](https://github.com/mcollina/help-me/releases/tag/v5.0.0) which [drops glob](https://github.com/mcollina/help-me/pull/19)
- [help-me was using glob v8](https://github.com/mcollina/help-me/issues/17) and glob < v9 has a dependency inflight which is no longer maintained and has [CWE-772: Missing Release of Resource after Effective Lifetime](https://github.com/isaacs/inflight/issues/19) [https://cwe.mitre.org/data/definitions/772.html](https://cwe.mitre.org/data/definitions/772.html) 

In summary: mqtt@5.3.4 › help-me@4.2.0 › glob@8.1.0 › inflight@1.0.4 (CWE-772)

If it helps, I would be happy to make a PR with help-me v5.0.0

**Comment by robertsLando** [comment]
@jackbridger Do it, thanks

**Comment by jackbridger** [comment]
> @jackbridger Do it, thanks

Thanks @robertsLando 

It's likely my own environment (I'm debugging) but just wanted to check in case I'm missing anything while running locally:

I'm getting a node mismatch where when I run:
node -v  (v20.10.0)
But when I run npm run test, I can see it's using v16.10.0. So it fails because node:test is not available 
```Error [ERR_UNKNOWN_BUILTIN_MODULE]: No such built-in module: node:test```

**Comment by robertsLando** [comment]
Node test runner requires node 18+. I was going to drop node 16 but I had to keep it as I didn't want to do a new major just for that, I have some other things in mind for next major
